executor/src/witgen/affine_expression.rs

```diff
@@ -576,6 +574,26 @@ mod test {
         input.iter().map(|x| (*x).into()).enumerate().collect()
     }
 
+    #[test]
+    pub fn test_affine_assign() {
```

Comment:
Can you also add a test where you call `solve` (on a copy) and then use the resulting in `assign` and check that the final expression is zero?